src/functionalTest/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/UpdateCaseTest.java

```diff
@@ -21,8 +21,17 @@
 
     @Test
     public void should_update_case() throws JSONException {
+        updateCaseAndVerifyResponse("updatecase/valid-request.json");
+    }
+
+    @Test
+    public void should_update_case_by_exception_record_with_auto_update_fields_without_warnings() throws JSONException {
+        updateCaseAndVerifyResponse("updatecase/valid-request-with-auto-case-update-fields.json");
+    }
+
+    private void updateCaseAndVerifyResponse(String s) throws JSONException {
```

Comment:
The parameter needs to have a meaningful name. I'd also add more parameters, e.g. expected response json.

Reply:
ok